src/Carousel/README.md

```diff
@@ -58,3 +62,31 @@ notes: |
   </Carousel.Item>
 </Carousel>
 ```
+
+### Theme Variables (SCSS)
```

Comment:
The theme variables are no longer necessary in the README with the [changes recently merged](https://github.com/edx/paragon/pull/931). Just a symptom of a delay in reviewing this PR, my apologies.